internal/application/manager.go

```diff
@@ -547,7 +547,8 @@ func (m *dataManager) ImportRecordedData(data *dtos.RecordedData, overwrite bool
 
 func (m *dataManager) uploadDevices(devices []coreDtos.Device, overwrite bool) error {
 	deviceClient := m.appSvc.DeviceClient()
-	for _, device := range devices {
+	for index := range devices {
+		device := devices[index]
```

Comment:
I assume this change was needed for the linter upgrade?

Reply:
Yep